frontend/Dockerstub

```diff
@@ -1,3 +1,6 @@
+RUN apt-get install --yes nginx
+RUN python3 -m pip install --upgrade aiohttp
```

Comment:
This `aiohttp` install can be removed since `ofrak` as of 2.1.0 has a dependency on `aiohttp`.

Reply:
Removed - with [3a537e8](https://github.com/redballoonsecurity/ofrak/pull/194/commits/3a537e843f3320898576f5104460ca9d932a36be) now the Python dependencies are installed in `base.Dockerfile` based on the requirements files.